### PR TITLE
apply timing related changes due to async backing on moonbase

### PR DESCRIPTION
**Pull request by eshaben**
### Description

Async backing is being enabled on moonbase alpha for runtime 2800, so this PR updates the timing-related changes.

### Checklist

- [x] I have added a label to this PR 🏷️
- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira

### After Translation Requirements

- [x] No additional PRs are required after the translations are done

### CODE REVIEW

File: variables.yml [hunk 1]
```diff
@@ -290,7 +290,7 @@ networks:
       collator_map_bond: 100
       max_candidates: 36
       round_blocks: 600
```
Comment (albertov19) [inline variables.yml:292]:
```suggestion
      round_blocks: 1200
```

File: variables.yml [hunk 2]
```diff
@@ -290,7 +290,7 @@ networks:
...
-      round_hours: 2
+      round_hours: 1
```
Comment (albertov19) [inline variables.yml:293]: Rounds are still 2 hours ->

image
```suggestion
      round_hours: 2
```

File: variables.yml [hunk 3]
```diff
@@ -321,26 +321,26 @@ networks:
     delegator_timings:
       leave_delegators:
         rounds: 2
-        hours: 4
+        hours: 2
```
Comment (albertov19) [inline variables.yml:324]:
```suggestion
        hours: 4
```

File: variables.yml [hunk 4]
```diff
@@ -321,26 +321,26 @@ networks:
...
       revoke_delegations:
         rounds: 2
-        hours: 4
+        hours: 2
```
Comment (albertov19) [inline variables.yml:327]:
```suggestion
        hours: 4
```

File: variables.yml [hunk 5]
```diff
@@ -321,26 +321,26 @@ networks:
...
       del_bond_less:
         rounds: 2
-        hours: 4
+        hours: 2
```
Comment (albertov19) [inline variables.yml:330]:
```suggestion
        hours: 4
```

File: variables.yml [hunk 6]
```diff
@@ -321,26 +321,26 @@ networks:
...
       rewards_payouts:
         rounds: 2
-        hours: 4
+        hours: 2
```
Comment (albertov19) [inline variables.yml:333]:
```suggestion
        hours: 4
```

File: variables.yml [hunk 7]
```diff
@@ -321,26 +321,26 @@ networks:
...
     collator_timings:
       max_offline:
         rounds: 2
-        hours: 4
+        hours: 2
```
Comment (albertov19) [inline variables.yml:337]:
```suggestion
        hours: 4
```

File: variables.yml [hunk 8]
```diff
@@ -321,26 +321,26 @@ networks:
...
       leave_candidates:
         rounds: 2
-        hours: 4
+        hours: 2
```
Comment (albertov19) [inline variables.yml:340]:
```suggestion
        hours: 4
```

File: variables.yml [hunk 9]
```diff
@@ -321,26 +321,26 @@ networks:
...
       can_bond_less:
         rounds: 2
-        hours: 4
+        hours: 2
```
Comment (albertov19) [inline variables.yml:343]:
```suggestion
        hours: 4
``

File: variables.yml [hunk 10]
```diff
@@ -375,7 +375,7 @@ networks:
       max_collators: Unlimited
       active:
         rounds: 3
-        hours: 6
+        hours: 3
```
Comment (albertov19) [inline variables.yml:378]:
```suggestion
        hours: 6
```

Please check how is this configured

**Review by albertov19 — CHANGES_REQUESTED** [review]
Check the rounds, for staking they were increased to 1200 blocks! 😄

File: variables.yml (hunk 1 above)
Comment (eshaben) [reply variables.yml:292]: ahhhhhh haha I double-checked everything in the configs but the most important thing 🤦‍♀️

File: learn/features/staking.md [hunk 11]
```diff
@@ -65,7 +65,7 @@ Some important parameters to understand in relation to the staking system on Moo
 === "Moonbase Alpha"
     |             Variable             |                                                                         Value                                                                         |
     |:--------------------------------:|:-----------------------------------------------------------------------------------------------------------------------------------------------------:|
-    |          Round duration          |                        {{ networks.moonbase.staking.round_blocks }} blocks ({{ networks.moonbase.staking.round_hours }} hours)                        |
+    |          Round duration          |                        {{ networks.moonbase.staking.round_blocks }} blocks ({{ networks.moonbase.staking.round_hours }} hour)                        |
```
Comment (eshaben) [inline learn/features/staking.md:68]:
```suggestion
    |          Round duration          |                        {{ networks.moonbase.staking.round_blocks }} blocks ({{ networks.moonbase.staking.round_hours }} hours)                        |
```

**Review by albertov19 — APPROVED** [review]
LGTM

**Pull request merged**